### PR TITLE
Converted to a currency control for consistency

### PR DESCRIPTION
**Pull request by tmccoy529**
For consistency converted the currency input component into a new component with default values. We only need to pass ID value and change events to it instead of other parameters that will always stay the same.

### CODE REVIEW

File: src/components/CurrencyControl.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import CurrencyInput from "react-currency-input";
```
Comment (martypowell) [inline src/components/CurrencyControl.jsx]: import ReactCurrencyInput

File: src/components/CurrencyControl.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const CurrencyControl = props => {
```
Comment (martypowell) [inline src/components/CurrencyControl.jsx]: CurrencyInput

Comment (martypowell) [reply src/components/CurrencyControl.jsx]: we can do this, to keep things simple, people don't need to know what currency input we are using.

File: src/components/CurrencyControl.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  const { value, id, onChangeEvent } = props;
+  return (
+    <React.Fragment>
```
Comment (martypowell) [inline src/components/CurrencyControl.jsx]: Don't think you need React.Fragment here, i don't believe it's doing anything.

File: src/components/CurrencyControl.jsx (hunk 1 above)
Comment (tmccoy529) [reply src/components/CurrencyControl.jsx]: done

File: src/components/CurrencyControl.jsx (hunk 3 above)
Comment (tmccoy529) [reply src/components/CurrencyControl.jsx]: removed

File: src/components/CurrencyControl.jsx (hunk 2 above)
Comment (tmccoy529) [reply src/components/CurrencyControl.jsx]: Done

**Pull request merged**